galaxy_api/api/migrations/0001_initial.py

```diff
@@ -0,0 +1,88 @@
+from django.db import migrations
+from django.db import models
+
+
+class Migration(migrations.Migration):
+    initial = True
+
+    dependencies = [("galaxy_auth", "0001_initial")]
+
+    operations = [
+        migrations.CreateModel(
+            name="Namespace",
+            fields=[
+                (
+                    "id",
+                    models.AutoField(
+                        auto_created=True,
+                        primary_key=True,
+                        serialize=False,
+                        verbose_name="ID",
+                    ),
+                ),
+                ("name", models.CharField(editable=False, max_length=64, unique=True)),
+                ("company", models.CharField(blank=True, max_length=64)),
+                ("email", models.CharField(blank=True, max_length=256)),
+                ("logo_url", models.CharField(blank=True, max_length=256)),
```

Comment:
Lets use `avatar_url` here, to try and maintain better compatibility with galaxy.